ui/src/app/applications/components/applications-list/applications-list.tsx

```diff
@@ -307,6 +307,15 @@ export const ApplicationsList = (props: RouteComponentProps<{}>) => {
     const [isAppCreatePending, setAppCreatePending] = React.useState(false);
     const loaderRef = React.useRef<DataLoader>();
     const {List, Summary, Tiles} = AppsListViewKey;
+    const [authSettings, setAuthSettings] = React.useState(null);
+
+    React.useEffect(() => {
```

Comment:
We already call services.authService.settings() in the top level App component. Can we set the context there instead?

Reply:
Agreed, the Context can be moved further up.